pyproject.toml

```diff
@@ -1,2 +0,0 @@
-[tool.black]
```

Comment:
Is there any particular reason to remove this file? This way you can just run `black .` instead of `black -l 120` and get the correct option